scipy/optimize/_linprog_util.py

```diff
@@ -641,8 +653,9 @@ def where(A):
                        "turn presolve off.")
             complete = True
             return (_LPProblem(c, A_ub, b_ub, A_eq, b_eq, bounds, x0),
-                    c0, x, undo, complete, status, message)
+                    c0, x, revstack, complete, status, message)
         # variables will equal upper/lower bounds will be removed later
+        # >> the statements below also change bounds and lp <<
```

Comment:
They don't anymore though, right? You created a copy.

Reply:
Yes. I'll remove the comments.